### PR TITLE
Added mobility derivatives to SplitCHWRes

### PR DESCRIPTION
**Pull request by tonkmr**
These changes address #5167. It makes the mobility a derivative material and accounts for its derivatives in the Jacobian.

### CODE REVIEW

**Comment by tonkmr** [comment]
@dschwen, can you review this?

**Comment by permcody** [comment]
Don't merge anything else until we unscrew the build statuses please. Just waiting for Yaqi.

File: modules/phase_field/src/kernels/SplitCHWRes.C [hunk 1]
```diff
@@ -10,24 +10,45 @@ InputParameters validParams<SplitCHWRes>()
...
+  // Iterate over all coupled variables
+  for (unsigned int i = 0; i < nvar; ++i)
+    _dmobdarg[i] = &getMaterialPropertyDerivative<Real>(_mob_name, _coupled_moose_vars[i]->name());
 }
 
 Real
 SplitCHWRes::computeQpResidual()
 {
-  return _mob[_qp] * _grad_u[_qp] * _grad_test[_i][_qp];
+  return _mob[_qp]*_grad_u[_qp]*_grad_test[_i][_qp];
```
Comment (dschwen) [inline modules/phase_field/src/kernels/SplitCHWRes.C]: :-/

**Comment by dschwen** [comment]
Looks fine to me. :+1:

**Pull request merged**